code/online_challenges/src/codechef/SLAB/slab.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Slab {
```

Comment:
Please reformat this file with 4 space indentation. Also call the file `Slab.java`.

Reply:
But i formatted it using https://codebeautify.org/javaviewer#

Reply:
What should I do!? Use a 4 space indendation or follow the formatting of online platform  https://codebeautify.org/javaviewer#. Plz suggest.

Reply:
@raksha009 Sorry. you are right, I didn't realize you can't change the settings there. Are you using an IDE? Try reformatting in that (For IntelliJ I know the option is there). If you want, I can reformat the file for you.

Reply:
Yes, I am using eclipse. I did 4 sapce indentation initially, will redo it again. Thanks